### ISSUE TITLE
multiple places define OD_OCP_SOURCE_APP_DOMAIN

### ISSUE THREAD

**Issue by kiwo** [opendevstack/ods-configuration-sample#41]
Hi, 

while trying to use tailor to setup a new jenkins in cd I ran into a problem missing new variables defined in the template but not nessesary in my (old) ods config. I found a definition for OD_OCP_SOURCE_APP_DOMAIN in ocp_project_config_source but it seems this is not the correct file to source to tailor apart from having to source multiple env files from different directories?!. The new templates.env.sample has now also got the definition for OD_OCP_SOURCE_APP_DOMAIN. 

Where should this be configured? If this is going to be configured centrally not in my referenced templates.env what would be best practice to include this config?

Best regards,

Chris

**Comment by clemensutschig** [comment]
@Michaelsauter can you help here?

**Comment by michaelsauter** [comment]
It could be that right now it needs to be configured in multiple places. I have a WIP pull request for a long time to unify this into one file, but progress has been slow. I hope I can pick it up again for 2.0.

I'd say we look over this on Friday. @henrjk do you have more insight on this? Maybe you came across this when you introduced Tailor into that part of ODS?

**Comment by henrjk** [comment]
Yes all variables needed by tailor via Tailorfile definitions in https://github.com/opendevstack/ods-project-quickstarters/tree/master/ocp-templates/ocp-config
are defined in the templates.env file you mentioned. 

I just noticed that in the template uploaded in 1.1.x the host is not defined 
in the route at:

https://github.com/opendevstack/ods-project-quickstarters/blob/b5c4f6ceb3356c67b48552460ac357403d0585d3/ocp-templates/templates/cd-jenkins-master.yml#L42-L56

whereas when replacing the upload templates with templates used for tailor I was ensuring that the tailor would not report a drift. I probably added the host because of that.
